.circleci/config.yml

```diff
@@ -84,6 +84,31 @@ references:
       path: test-reports
       destination: test-reports
 
+  benchmark: &benchmark
+    run:
+      name: Run benchmarks
+      command: |
+          pip install --user gitpython
+          echo 'export PYTHONPATH=$PYTHONPATH:.' >> $BASH_ENV
+          source $BASH_ENV
+          BENCHSCR="benchmarks/benchmarker.py"
+          echo "Running benchmark on current HEAD"
+          python "$BENCHSCR" run -c
+          echo "Comparing with orign/master"
+          git reset --hard origin/master
+          echo "PYVER=${PYVER}"
+          rm freud/*.cpp
+          python${PYVER} setup.py build_ext --inplace --COVERAGE --ENABLE-CYTHON
+          git checkout "${CIRCLE_SHA1}" -- benchmarks/
+          echo "Running benchmark on origin/master"
+          python "$BENCHSCR" run -c
+          python "$BENCHSCR" compare origin/master "${CIRCLE_SHA1}"
+
+  store_benchmarks: &store_benchmarks
+    store_artifacts:
+      path: benchmarks/reports
+      destination: benchmarks-reports
```

Comment:
The current output on CircleCI for "Uploading artifacts" says:
```
Uploading /home/ci/ci/freud/benchmarks/reports to benchmarks-reports
No artifact files found at /home/ci/ci/freud/benchmarks/reports
```
I think `path` has to point to a specific file - please refer to the CircleCI docs and figure out how to upload all the result files generated by the benchmarks.